.travis.yml

```
@@ -1,17 +1,14 @@
 language: php
 
 php:
-  - 7.1
-  - 7.2
-  - 7.3
-  - 7.4
+  - 8.0
 
 before_script:
   - composer update
 
 script:
   - ./vendor/bin/phpunit --disallow-test-output --coverage-clover ./clover.xml
-  - if php -i |grep -qE xdebug; then ./vendor/bin/infection --min-msi=100 --min-covered-msi=100 --threads=4; fi
+  - if php -i |grep -qE xdebug; then -d xdebug.mode=coverage ./vendor/bin/infection --min-msi=100 --min-covered-msi=100 --threads=4; fi
```

This *might* fix the build failure?
```suggestion
  - if php -i |grep -qE xdebug; then php -d xdebug.mode=coverage ./vendor/bin/infection --min-msi=100 --min-covered-msi=100 --threads=4; fi
```

---

I didn't notice that I forgot to put php in front of it. Oeps.